attest/eventlog.go

```diff
@@ -31,6 +31,17 @@ import (
 	"github.com/google/go-tpm/tpmutil"
 )
 
+// ReplayError describes the parsed events that failed to verify against
+// a particular PCR.
+type ReplayError struct {
+	BadEvents   []Event
+	InvalidPCRs []int
+}
+
+func (e ReplayError) Error() string {
```

Comment:
nit: for lint please add a godoc comment here

Reply:
Done.